core/store/src/db/refcount.rs

```diff
@@ -38,6 +38,9 @@ pub fn decode_value_with_rc(bytes: &[u8]) -> (Option<&[u8]>, i64) {
         debug_assert!(bytes.is_empty());
         return (None, 0);
     }
+    // TODO(mina86): Use rsplit_array_ref once split_array feature is stabilised
+    //    let (head, tail) = bytes.rsplit_array_ref<8>();
+    //    let rc = i64::from_le_bytes(tail);
```

Comment:
I am tempted to add a `near-std` crate with polyfills for stuff we are waiting to stabilize, and for other std-only utilities, like https://github.com/near/nearcore/blob/master/runtime/near-vm-logic/src/array_utils.rs.